src/encoder/iiif_encoder.rs

```diff
@@ -106,6 +109,7 @@ impl TileSaver for IIIFTileSaver {
         let image_path = image_dir_path.join(filename);
         debug!("Writing tile to {:?}", image_path);
         std::fs::create_dir_all(&image_dir_path)?;
-        tile.image.save(image_path).map_err(image_error_to_io_error)
+	let mut file = File::create(&image_path)?;
```

Comment:
Shouldn't we wrap that in a BufWriter for performance ?

Reply:
I'm afraid that I'm not familiar with Rust, and have just hacked the changes by ear and by looking at Rust library documentation on the web.  If you feel that changes can be made to improve performance then go for it!

Reply:
You can look at how it's done in the normal jpeg dezoomer: https://github.com/lovasoa/dezoomify-rs/blob/master/src/encoder/canvas.rs#L71-L73

It's really just one line to add to wrap the File object in a BufWriter.